tier2/checklist.md

```diff
@@ -126,7 +126,8 @@ Below is a list of suggested tools to run for code analysis:
 | --------------- | -------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Repo Linter     | Lint repositories for common issues such as missing files,etc...                       | https://github.com/todogroup/repolinter                                                                                                                                                                                       |
 | Gitleaks        | Protect and discover secrets using Gitleaks 🔑                                         | https://github.com/gitleaks/gitleaks <br /> Use and run [gitleaks.yml](https://github.com/DSACMS/repo-scaffolder/blob/main/tier2/%7B%7Bcookiecutter.project_slug%7D%7D/.github/workflows/gitleaks.yml) provided in repository |
-| git filter-repo | Entirely remove unwanted files / files with sensitive data from a repository's history | https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository                                                                                                      |
+| git filter-repo | Entirely remove unwanted files / files with sensitive data from a repository's history | https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository     |
+| Bulk_extractor | Check for secrets, URLs, emails, etc. | https://github.com/simsong/bulk_extractor https://docs.google.com/document/d/1ENQzI3ea3o9izOPu0sghwk1FU8TReVoCHlt21NoIKZQ/edit?tab=t.0#heading=h.4wtx3g49kbj0  |
```

Comment:
Since external users won't have access to google docs, I would change this link to the ospo-guide doc!

Reply:
Changed to point to guide.